TradeItIosTicketSDK2/TradeItSDK.swift

```diff
@@ -0,0 +1,21 @@
+@objc public class TradeItSDK: NSObject {
```

Comment:
@mitochondrion pls do this. I know it makes no sense. Literally just check it out and run it and see if you get any compiler errors.